src/challenges.js

```diff
@@ -5,25 +5,56 @@
 const each = (elements, cb) => {
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
+  return elements.forEach(cb);
 };
 
 const map = (elements, cb) => {
   // Produces a new array of values by mapping each value in list through a transformation function.
   // Return the new array.
+  return elements.map(cb);
+
+  // let myMap = [];
+  // for (let i = 0; i < elements.length; i++) {
+  //   myMap = elements[i].push(myMap);
```

Comment:
Instead of pushing into `elements[i]` (which is probably a string or number), you should push into `myMap` also you should be pushing `elements[i]` after it was passed into `cb`.